.gitmodules

```diff
@@ -9,4 +9,4 @@
 	url = https://github.com/setprotocol/set-protocol-v2
 [submodule "lib/index-coop-smart-contracts"]
 	path = lib/index-coop-smart-contracts
-	url = https://github.com/setprotocol/index-coop-smart-contracts
+	url = https://github.com/ncitron/index-coop-smart-contracts
```

Comment:
Yeah you can see the branch that I am using here: https://github.com/ncitron/index-coop-smart-contracts/tree/coop-tax

It just changes the `startRebalanceWithUnits` function of `GIMExtension` to virtual so it can be overridden in `ApeRebalanceExtension`